versionutils/repo.go

```diff
@@ -135,6 +136,7 @@ func parseTomlFromDir(relativeDir, configType string) ([]*toml.Tree, error) {
 type TomlWrapper struct {
 	Overrides   []*toml.Tree
 	Constraints []*toml.Tree
+	SoloIo      []*toml.Tree
```

Comment:
don't need this with the multiple toml file approach
(had initially thought to overload the `Gopkg.toml` file)